src/main/java/com/CezaryZal/manager/builder/TokenBuilder.java

```diff
@@ -0,0 +1,32 @@
+package com.CezaryZal.manager.builder;
+
+import com.CezaryZal.config.SecurityConstants;
+import com.CezaryZal.entity.User;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.stereotype.Service;
+
+import java.util.Date;
+
+@Service
+public class TokenBuilder {
+
+    public String buildTokenByUser(User foundUser){
```

Comment:
a co TokenBuildera interesuje co to za user i skąd on się wział, jemu to bez różnicy czy on jest znaleziony, czy się sam przypałętał, jego to wali (zmieniłbym nazwe z foundUser na user) :)

Reply:
ok :P
Chciałem zprecyzować co do za user